### DISCUSSION TITLE
<title>problème dans le déploiement sur github page

### DISCUSSION THREAD

**Discussion by eziopelle** [community/community#113650] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello, 

Je veux rendre un fichier HTML index.html disponible sur internet de façon simple avec une url par défaut. 

Cependant mon github page bloque sur l'url "eziopelle-garden.me" qui fait référence a un ancien projet. Quand je veux rien mettre dans la partie domaine personnalisé le save est grisé. 

Svp aidez moi
<img width="989" alt="Capture d’écran 2024-03-22 à 10 14 41" src="https://github.com/community/community/assets/56078363/ad103f0a-1558-441b-b003-004eb1cc103c">

**Comment by Akash1134** [comment]
Hi @eziopelle , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
